Tests/SDFCoreTests/MacroInitializerSorterTests.swift

```diff
@@ -94,4 +116,40 @@ struct TestData {
     }
     """
   }
+
+  enum WithNestedDefinitions: String, CaseIterable {
+    case actorContent = """
+    @TestMacro
+    actor Outer {
+      actor Some {
+        init(b: String,
+             a: Int) {}
+      }
+      init(b: String, 
+           a: Int) {}
+    }
+    """
+    case classContent = """
+    @TestMacro
+    class Outer {
+      class Some {
+        init(b: String,
+             a: Int) {}
+      }
+      init(b: String, 
+           a: Int) {}
+    }
+    """
+    case structContent = """
+    @TestMacro
+    struct Outer {
+      struct Some {
+        init(b: String,
```

Comment:
I'd rename these parameters so they're different from the Outer initializer, so it's more clear when we're asserting in the tests. Something ultra explicit like `init(someB: String, someA: Int)` would be great.